tests/keras/activations_test.py

```diff
@@ -42,6 +69,18 @@ def softmax(values):
     assert_allclose(result, expected, rtol=1e-05)
 
 
+def test_softmax_invalid():
+    '''
+    Test for the expected exception behaviour on invalid input
```

Comment:
Sorry for these tidiness problems, I will try to reduce the effort for you to review in future.